### ISSUE TITLE
[Camera] Image capture is too slow on Android

### ISSUE THREAD

**Issue by DelphiWorlds** [DelphiWorlds/Kastri#25]
Time between capturing an image and it appearing as a still is around 1 second, which is way too slow.

**Comment by alissonrodrigo** [comment]
Thank you all questions have been answered.
I don't know if here would be the place to ask this, but I'm studying your library right now and it's amazing. In the camera demo when I take a picture there is a deley of at least 2 seconds before showing the taken picture. i'm trying to create a wait message with the frozen photo, could you tell me which way i can go for this?
Grateful

**Comment by alissonrodrigo** [comment]
Hi, has the problem regarding image capture delay been fixed? Grateful

**Comment by DelphiWorlds** [comment]
Unfortunately, no.. I am also planning to move the camera implementation to use CameraX, which can be supported only in Delphi 11 or higher.

**Comment by alissonrodrigo** [comment]
> Infelizmente, não. Também estou planejando mover a implementação da câmera para usar o CameraX, que pode ser compatível apenas com Delphi 11 ou superior.

Yes there is, but do you have any projections regarding availability time? Grateful

**Comment by alissonrodrigo** [comment]
Do you have any news about the Camera?

**Comment by DelphiWorlds** [comment]
No, other than that I have [a project here](https://github.com/DelphiWorlds/Playground/tree/main/Demos/CameraX) that is a test app for CameraX, however as per the readme, I have not been able to make that work.

**Comment by alissonrodrigo** [comment]
Ah yes, I follow this project! if you can make it work and can share I would be very grateful

**Comment by DelphiWorlds** [comment]
Linked project is here: https://github.com/DelphiWorlds/Playground/tree/main/Demos/SimpleCamera

It has an explanation regarding slowness of image capture (as a bitmap)

**Comment by DelphiWorlds** [comment]
Closing all camera related requests - any future work will need to be funded
